src/App.js

```diff
@@ -19,7 +18,7 @@ class App extends Component {
         <div>
           <Banner />
           <Header />
-          <section className="usa-section usa-grid">
+          <section>
```

Comment:
If we don't want to use `usa-grid` maybe we can start moving towards http://flexboxgrid.com?